projects/client/src/lib/sections/lists/SeasonList.svelte

```diff
@@ -92,11 +92,3 @@
     {/if}
   {/snippet}
 </ShadowList>
-
-<style>
-  :global(.shadow-list-container) {
-    :global(.trakt-dropdown-list-container[data-size="small"]) {
-      margin-right: var(--ni-neg-36);
-    }
-  }
-</style>
```

Comment:
@seferturan this changes spacing and UX, class should be here.

That warning is added by the sveltekit compiler since the last upgrade saying we should move these global overrides to `.css` files and do `import './style.css`.

Reply:
To clarify:

```svelte
<style>
    :global(.selector) {
       /* This results in warning. */
    }
</style>

<style>
    .parent {
        :global(.selector) {
           /* This is considered okay since it's namespaced. */
        }
    }
</style>
```

To be honest if you ask me this is technically an "optimization" for the dev-server to be "faster". I would see if we could silence this warning TBH, I find it easier to follow the styles alongside the component (even if applied globally).

What are your thoughts @seferturan?